File: Solution_Practice_I/java/prob1_1.java
import java.util.Scanner;
public class prob1_1 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int seconds;
        do{
            System.out.println("Ingresa los segundos");
            seconds = input.nextInt();
        }while(seconds<0);
        int hours = seconds / (3600);
        int minutes = (seconds % 3600) / 60;
        seconds %= 60;
        System.out.println(
            (hours < 10 ? "0" + hours : hours) + ":" + 
            (minutes < 10 ? "0" + minutes : minutes) + ":" + 
            (seconds < 10 ? "0" + seconds : seconds)
        );
        input.close();
    }
}

File: Solution_Practice_I/python/prob1_1.py
while True:
    seconds = int(input("Ingrese los segundos: "))
    if(seconds>=0): 
        break
hours= seconds // 3600
minutes = (seconds%3600)//60
seconds%=60
print(
    (f"0{hours}:" if hours<10 else f"{hours}:")+
    (f"0{minutes}:" if minutes<10 else f"{minutes}:")+
    (f"0{seconds}" if seconds<10 else f"{seconds}")
)